lab-kyle/lib/fs-function.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
This is a good start but there is an issue with the way you're console logging your error, see my comment below.